source/features/patch-diff-links.tsx

```diff
@@ -3,10 +3,12 @@ import select from 'select-dom';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
+import {getCleanPathname} from '../github-helpers';
 
 function init(): void {
-	let commitUrl = location.pathname.replace(/\/$/, '');
+	let commitUrl = '/' + getCleanPathname();
 
+	// Avoids a redirection
```

Comment:
Adding this comment to indicate that the normalization is not necessary anymore, in contradictory to #104.